src/types/records-types.ts

```diff
@@ -27,6 +27,8 @@ export type RecordsWriteDescriptor = {
 
 export type RecordsWriteReply = GenericMessageReply;
 
+export type RecordsDeleteReply = GenericMessageReply;
```

Comment:
One of the main goals for #289 is to ensure the return type is not generic and very specific to the message being handled.

See type `RecordsReadReply` that demonstrates this intent. Line 28 in this PR ought to be updated accordingly also, but that can be optional it being outside the context of this particular PR.

Reply:
Thanks @thehenrytsai. `RecordsReadReply` seems to be an extension of type `GenericMessageReply`:

```
export type RecordsReadReply = GenericMessageReply & {
  record?: RecordsWriteMessage & {
    data: Readable;
  }
};
````

If I understand your comment correctly,`RecordsWriteReply` and `RecordsDeleteReply` also similarly need to be extended types (so that they're more Write- and Delete-specific).  If yes, where can I find the specific data models for these types so I know what fields they need to include?  I didn't find them on [decentralized-web-node/spec/#response-objects](https://identity.foundation/decentralized-web-node/spec/#response-objects).

Reply:
@shobitb, you are correct that the existing DWN spec is behind. The best way I can think of is to just look at the handler method, which I just realized after checking that `GenericMessageReply` is sufficient. I'd rather for us not introduce new types (`RecordsWriteReply` and `RecordsDeleteReply`)  unnecessarily in these two particular cases. Let's remove those two types and use `GenericMessageReply` directly and we can get this approved and merged!

Reply:
Actually, let's just merge this in. I can make those changes if needed! Really appreciate your contributions!